imgp.1

```diff
@@ -55,6 +55,9 @@ Output resolution in HRESxVRES or percentage.
 .BI "-o, --rotate=" deg
 Rotate clockwise by a specified angle (in degrees). Negative inputs rotate anti-clockwise. Rotation by 0 degree is not allowed.
 .TP
+.BI "-q, --quality=" N
+Save the image with a specified quality factor N. Only JPEG image support for this feature is present in PIL currently. Default value is 75.
```

Comment:
`Only JPEG image support for this feature is present in PIL currently.`

change to

`JPEG-only`.